lib/quicly.c

```diff
@@ -1573,6 +1620,13 @@ static int on_ack_stream_id_blocked(quicly_conn_t *conn, int acked, quicly_ack_t
     return 0;
 }
 
+static int on_ack_cc(quicly_conn_t *conn, int acked, quicly_ack_t *ack)
+{
+    conn->egress.cc.this_ack.nsegs += 1;
+    conn->egress.cc.this_ack.nbytes += ack->data.cc.bytes_in_flight;
```

Comment:
If these variables are supposed to be what's newly acked by a received ack, I don't think this is correct.